docs/root/intro/arch_overview/service_discovery.rst

```diff
@@ -91,6 +91,15 @@ Generally active health checking is used in conjunction with the eventually cons
 discovery service data to making load balancing and routing decisions. This is discussed further in
 the following section.
 
+.. _arch_overview_service_discovery_types_eds:
+
+Endpoint discovery service (EDS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The *endpoint discovery service* has the same role as SDS in the v1 API, the new name better describes what the API does
```

Comment:
We now have the joy of Secret Discovery Service also being SDS. How about you just update the SDS docs above to only use the EDS terminology? Since the old SDS is deprecated, nobody should be writing new configs referring to it.

Reply:
Good point. I'll change the description to only use the EDS terms.

And I'm also planning to change the order of items. EDS first, then v1 SDS. Probably we still should have v1 SDS section on docs?

Reply:
I tried describing EDS without SDS words. I pick up some descriptions from previous SDS section wihch are still valid in EDS world as far as I know.